Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py

```diff
@@ -0,0 +1,708 @@
+                                                    isBundle=res_json.get('is_bundle'),
+                                                    ratingIndustryMedian=res_json.get('rating_industry_median'),
+                                                    primaryCompany=res_json.get('primary_company'),
+                                                    permissions=res_json.get('permissions'),
+                                                    isPrimary=res_json.get('is_primary'),
+                                                    securityGrade=res_json.get('security_grade'),
+                                                    inSpmPortfolio=res_json.get('in_spm_portfolio'),
+                                                    isMycompMysubsBundle=res_json.get('is_mycomp_mysubs_bundle'),
+                                                    companyFeatures=res_json.get('company_features')
+                                                    )
+    else:
+        res_json = res.json()
+        generic_iam_context = OutputContext_Details(errorCode=res.status_code,
+                                                    errorMessage=res_json)
+
+    generic_iam_context_dt = f'{generic_iam_context.command}(val.id == obj.id && val.instanceName == obj.instanceName)'
+
+    outputs = {
+        generic_iam_context_dt: generic_iam_context.data
+    }
+
+    readable_output = tableToMarkdown(name='Get Company Details:',
+                                      t=generic_iam_context.data,
+                                      headers=["errorCode", "errorMessage", "guid", "customId", "name", "description",
+                                               "ipv4Count", "peopleCount", "shortName", "industry", "industrySlug",
+                                               "subIndustry", "subIndustrySlug", "homePage", "primaryDomain", "type",
+                                               "displayURL", "ratingDetails", "ratings", "searchCount",
+                                               "subscriptionType", "sparkline", "subscriptionTypeKey",
+                                               "subscriptionEndDate", "bulkEmailSenderStatus", "serviceProvider",
+                                               "customerMonitoringCount", "availableUpgradeTypes", "hasCompanyTree",
+                                               "hasPreferredContact", "isBundle", "ratingIndustryMedian",
+                                               "primaryCompany", "permissions", "isPrimary", "securityGrade",
+                                               "inSpmPortfolio", "isMycompMysubsBundle", "companyFeatures"],
+                                      removeNull=True
+                                      )
+    return (
+        readable_output,
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def get_company_findings_command(client, args, first_seen=None, fetch_incidents=False):
+    if fetch_incidents:
+        guid = args.get('guid', None)
```

Comment:
demisto.params was passed is passed as parameter to method from fetch-incidents, which was referred with parameter name 'args', Since this method is executed from fetch-incident also from command.